R/calcProjectPipelines.R

```diff
@@ -0,0 +1,66 @@
+#' calc Project Pipelines
+#'
+#' Calculate the expected near-term deployment of technologies
+#'
+#' @author Pascal Weigmann
+#'
+#' @param subtype choose technology, either `CCS`, `Nuclear`
+#'
+#' @export
+calcProjectPipelines <- function(subtype) {
+  if (subtype == "CCS") {
+    x <- readSource("IEA_CCUS", subtype = "pipeline")
```

Comment:
This section is redundant to`calcCCScapacity` capacity. I assume you looked into removing the redundancy, but did not find an easy way to do so? If that is the case, then it is fine by me.

Reply:
Edit: just read you comment concerning this. I am fine with the redundancy in this case. It gives you flexibility, should requirements by the different teams diverge in the future.

Reply:
Agreed, it is not great, ideally the two approaches will be merged in the future.